pkg/bootstrapper/bootstrapper_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-//cniTest holds the location of the directories and files required for running some of the CNI tests
+// cniTest holds the location of the directories and files required for running some of the CNI tests
```

Comment:
Formatting  fixes shouldn't be done in a bug fix commit. This has been called out a few times.